packages/apps/graph-client/src/pages/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { useGetBlocksSubscription } from '../__generated__/sdk';
+import { ChainwebGraph } from '../components/chainweb';
+import { Text } from '../components/text';
+import { styled } from '../styles/stitches.config';
+import { useParsedBlocks } from '../utils/hooks/useParsedBlocks';
+
+import Head from 'next/head';
+import React, { useEffect } from 'react';
+
+const StyledMain: any = styled('main', {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  flexDirection: 'column',
+  my: '5rem',
+});
+
+export default function Home(): JSX.Element {
+  const { loading, data } = useGetBlocksSubscription();
+  const dataString = JSON.stringify(data);
```

Comment:
usePrevious or https://github.com/kentcdodds/use-deep-compare-effect
to prevent performanceloss, this hook has a memoize function within it ,preventing this from happening on each rerender.

Reply:
useDeepCompareEffect didn't work since data can be undefined (there is an open issue on this) so I opted for usePrevious + lodash.isEqual comparison